### PR TITLE
Update PM UI README rendering spec

### PR DESCRIPTION
**Pull request by jgonz120**
Update to specify new message and tab selection behavior.

### CODE REVIEW

File: accepted/2024/PMUI-Readme-rendering.md [hunk 1]
```diff
@@ -22,15 +22,29 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
 
+The selected tab will be maintained between sessions and when switching packages.
```
Comment (donnie-msft) [inline accepted/2024/PMUI-Readme-rendering.md]: Is this implemented already or is a PR coming?

File: accepted/2024/PMUI-Readme-rendering.md [hunk 2]
```diff
@@ -22,15 +22,29 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
 
+The selected tab will be maintained between sessions and when switching packages. 
+If the README tab is removed, then the "Package Details" tab will automatically selected.
```
Comment (donnie-msft) [inline accepted/2024/PMUI-Readme-rendering.md]: This isn't a change from what's already shipped, right?

**Review by donnie-msft — APPROVED** [review]

File: accepted/2024/PMUI-Readme-rendering.md [hunk 3]
```diff
@@ -22,15 +22,20 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
...
 ![Alt text](../../meta/resources/ReadMePMUI/NoReadmeTab.png)
 
 #### README Tab
 This tab is rendered if a remote source available to download the readme or the Installed tab is selected and the selected version is found in the global packages folder.
 This is to ensure that users using the Browse tab will not see the README tab if they do not have a remote source that allows README downloads.
 
-If the README tab is rendered and we do not find a README then a message is rendered encouraging the user to contact the package author to upload a README.
-![alt text](../../meta/resources/ReadMePMUI/NoReadMeFound.png)
+When there is no README available we will display a message to the user in the README tab.
+>There is no README available for the selected package version. For me information please visit [aka.ms/nuget/noreadme](https://aka.ms/nuget/noreadme).
```
Comment (Nigusu-Allehu) [inline accepted/2024/PMUI-Readme-rendering.md]: LGTM. Assuming the aka.ms link will be linked to its own page in the future

**Review by Nigusu-Allehu — APPROVED** [review]

File: accepted/2024/PMUI-Readme-rendering.md (hunk 2 above)
Comment (nkolev92) [reply accepted/2024/PMUI-Readme-rendering.md]: What does remove mean in this case?
How do you remove the tab?

File: accepted/2024/PMUI-Readme-rendering.md [hunk 4]
```diff
@@ -22,15 +22,20 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
 The README will be available in all the PM UI tabs.
 It will also be displayed for both the solution level and project level package managers.
```
Comment (nkolev92) [inline accepted/2024/PMUI-Readme-rendering.md:24]: What about which tab we default to?

File: accepted/2024/PMUI-Readme-rendering.md (hunk 1 above)
Comment (nkolev92) [reply accepted/2024/PMUI-Readme-rendering.md]: Does any of this consider whether the source supports readmes?

**Review by nkolev92 — COMMENTED** [review]
Is this intended to address the `default tab` feedback?

File: accepted/2024/PMUI-Readme-rendering.md (hunk 1 above)
Comment (jgonz120) [reply accepted/2024/PMUI-Readme-rendering.md]: > Is this implemented already or is a PR coming?

Already implemented, just clarifying that even if the selected package does not have a README we will stay on the tab.

Comment (jgonz120) [reply accepted/2024/PMUI-Readme-rendering.md]: > Does any of this consider whether the source supports readmes?

The start of the section says "This tab is rendered if a remote source available to download the readme or the Installed tab is selected and the selected version is found in the global packages folder." I feel like that covered it.

File: accepted/2024/PMUI-Readme-rendering.md (hunk 3 above)
Comment (jgonz120) [reply accepted/2024/PMUI-Readme-rendering.md]: yup! @OliaG  is working on adding a section to the NuGet FAQs

File: accepted/2024/PMUI-Readme-rendering.md (hunk 4 above)
Comment (jgonz120) [reply accepted/2024/PMUI-Readme-rendering.md:24]: I added a line about that.

File: accepted/2024/PMUI-Readme-rendering.md (hunk 2 above)
Comment (jgonz120) [reply accepted/2024/PMUI-Readme-rendering.md]: > This isn't a change from what's already shipped, right?

Correct, just describing what's already implemented. 

> What does remove mean in this case? How do you remove the tab?

I updated the language to say "available". The availability of the tab is described in the README Tab section.

File: accepted/2024/PMUI-Readme-rendering.md [hunk 5]
```diff
@@ -22,18 +22,54 @@ The PM UI will be updated to have tabs for the Package Details and the the READM
...
+---
+flowchart TD
+    A(Which PM UI Tab Selected?)
+    A --> |Browse|B(Source supports downloading README?)
+    B --> |Yes|C(README tab visible)
+    B --> |No|D(README tab hidden)
+    A --> |Install/Update/Consolidate|E(Source supports downloading README?)
+    E --> |Yes|C
+    E --> |No|F(Selected package version is in GPF?)
+    F --> |Yes|C
```
Comment (nkolev92) [inline accepted/2024/PMUI-Readme-rendering.md:55]: I'm still of the opinion that: 

Source supports downloading readme -> No _> Selected package version is in GPF? -> Yes -> Readme Tab visible 

should be

Source supports downloading readme -> No _> Selected package version is in GPF? -> Yes -> Does package have readme? -> Yes - Readme Tab Visible.

The default view should be what the package itself has. 

The feedback was the readme tab selection should be readme existence aware. 

Are we explicitly deciding not to take that into account? 
The tab chosen should be what's most helpful for the user, and I don't think it is if we show an empty readme tab by default when we know that the readme isn't available

**Pull request merged**